source/transactions/tests/auto-start.yml

```diff
@@ -312,7 +310,7 @@ tests:
               level: "snapshot"
             lsid: session0
             txnNumber:
-              $numberLong: "2"
```

Comment:
Should the initial no-op commit still increment the transaction number on the client? I know this is an implementation detail but for the sake of the tests we should decide.

Reply:
The spec says about startTransaction:
```
Drivers MUST increment the txnNumber for the corresponding server session and reset the stmtId for this transaction to zero.
```

So txnNumber should be 2 here.